examples/layer_wise_benchmarks/parse.py

```
@@ -262,6 +238,22 @@ def shortest_common_supersequence(a, b):
 
 conn.close()
 
+# Check ambiguous modules
+if args.module:
+    for problem in problem_set:
+        num_matches_per_run = [0] * (len(problem["runs"]) + 1)
+        for i, ((range_start, _, text), kernel_count) in enumerate(
```

Ruff (0.14.10)

Loop control variable `i` not used within loop body

(B007)